snooty/rstspec.toml

```
@@ -1600,6 +1600,10 @@ type = {link = "https://www.mongodb.com/docs/atlas/reference/api-resources-spec/
 [role."oas-atlas-op"]
 type = {link = "https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#operation%s"}
 
+[role."atlascli"]
+help = """Link to a page in the Atlas CLI documentation."""
+type = {link = " [https://www.mongodb.com/docs/atlas/cli/stable%s] ", ensure_trailing_slash = true}
```

```suggestion
type = {link = "https://www.mongodb.com/docs/atlas/cli/stable%s", ensure_trailing_slash = true}
```
there's some weird brackets in there, this should work better

---

Oops, yes, paste mishap. Fixed! Thank you!